src/types/ProposalFieldValue.ts

```diff
@@ -9,6 +9,7 @@ export interface ProposalFieldValue {
 	value: string;
 	createdAt: Date;
 	applicationFormField?: ApplicationFormField;
+	isValid: boolean;
```

Comment:
Whoops I hit "submit" too early -- we have a funny thing here -- isValid *is* internally writable, but from an API perspective this is readonly (we don't want clients telling us if something is valid or not).

There is a pattern for this over here, where we have an `InternallyWritableBulkUpload` and a `WritableBulkUpload`: https://github.com/PhilanthropyDataCommons/service/blob/main/src/types/BulkUpload.ts

You'll want an `InternallyWritableProposalFieldValue` and a `WritableProposalFieldValue`